is_core/contrib/background_export/tasks.py

```diff
@@ -101,12 +101,16 @@ def background_serialization(self, exported_file_pk, rest_context, language, req
 
         # Perform user-defined verification before saving exported file into the database.
         if settings.BACKGROUND_EXPORT_TASK_CALLBACK:
-            import_string(settings.BACKGROUND_EXPORT_TASK_CALLBACK)(
-                request=request,
-                queryset=queryset,
-                filename=filename,
-                exported_file=exported_file,
-            )
+            try:
+                import_string(settings.BACKGROUND_EXPORT_TASK_CALLBACK)(
+                    request=request,
+                    queryset=queryset,
+                    filename=filename,
+                    exported_file=exported_file,
+                )
+            except tuple(import_string(exc) for exc in settings.BACKGROUND_EXPORT_TASK_HANDLED_EXCEPTIONS) as exc:
```

Comment:
I know that even the callback functionality is missing tests, but maybe it is time to add some. I'll leave this to your decision.

Reply:
Hmm, I agree that tests would be useful here, however, adding them would be really cumbersome. The whole background export functionality is not tested, not just the callback. Apart from that, setting up tests locally is not particularly easy in this project.

I'll create a separate issue with a proper time estimate for the near future.

Reply:
Yeah, I know, it would require some solid effort to get it working. Agreed.